system/settings/settings.xml

```diff
@@ -1821,38 +1821,64 @@
     <category id="upnp" label="20187" help="36322">
       <requirement>HAS_UPNP</requirement>
       <group id="1" label="16000">
-        <setting id="services.upnpserver" type="boolean" label="21360" help="36323">
+        <setting id="services.upnp" type="boolean" label="1270">
           <level>0</level>
           <default>false</default>
           <control type="toggle" />
         </setting>
+        <setting id="services.upnpserver" type="boolean" label="21358" help="36323">
+          <level>1</level>
```

Comment:
Hm maybe you are misinterpreting the term `level`. In this case it means the setting level visibility i.e.
* 0 = Basic
* 1 = Standard
* 2 = Advanced
* 3 = Expert
* 4 = Internal

If you want to have the setting indented under the general "Enable UPnP" setting you need to add the `parent="services.upnp"` attribute on the `<setting>` tag.